[DCOS-57225] Add docs for DSE 3.1.0-6.7.6 release (#2554)
mesosphere/dcos-docs-site: pages/mesosphere/dcos/services/dse/3.1.0-6.7.6/release-notes/index.md

  - [0.56.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.1)
  - [0.56.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.0)
  - [0.55.5](https://github.com/mesosphere/dcos-commons/releases/tag/0.55.5)

## New Features
- Added support for DSS [volume profiles](/mesosphere/dcos/services/{{ model.serviceName }}/latest/configuration/#using-volume-profiles)
- Added [custom domain](/mesosphere/dcos/services/{{ model.serviceName }}/latest/security/#forwarding-dns-and-custom-domain) support

## Improvements
- Remove non-HTTPS resources and references by replacing them with HTTPS ones
- Added more directories to the PATH (resources/cassandra/bin, resources/cassandra/tools/bin)
- Added descriptions for configuration parameters
- Updated UI URLs to redirect to correct reference doc pages
- Exclude {{ model.techOpsName }} keyspace from list of keyspaces managed by Repair Services
- Accurate Scheduler Plan status in the UI

## Bug Fixes
- Used correct YAML markup for TPC-related configurations: TPC CORES, TPC IO CORES, IO GLOBAL QUEUE DEPTH
- DSEFS could be enabled independently from DSE Analytics

> - Ignore error code of nodetool commands, stop command to run indifinitely.

Reviewer: ```suggestion
- Ignore error code of `nodetool` commands; stops command from running indefinitely.
```

> - Save OpsCenter event-plugins on volume, Alert sending configuration are now persisted between OpsCenter restarts.

Reviewer: ```suggestion
- Fixed OpsCenter event-plugins not being saved and getting lost on restart. Now, alert-sending configurations are now persisted between OpsCenter restarts.
```

> - Fix OpsCenter UI shows gray ring on restart

Reviewer: ```suggestion
- Fixed OpsCenter UI from showing a gray ring after a restart
```
